### PR TITLE
Fixup base model: missing multiheading in GAT and MFC

### PR DESCRIPTION
**Pull request by streeve**

### CODE REVIEW

File: models/GATStack.py [hunk 1]
```diff
@@ -60,5 +60,7 @@ def __init__(
             Linear(25, output_dim),
         )
 
+        super()._multihead(input_dim, output_dim, num_nodes, num_shared)
```
Comment (pzhanggit) [inline models/GATStack.py:63]: Remove the self.mlp part in line 55-61?

File: models/MFCStack.py [hunk 2]
```diff
@@ -50,5 +50,7 @@ def __init__(
             Linear(25, output_dim),
```
Comment (pzhanggit) [inline models/MFCStack.py:50]: Same here. self.mlp is part of the head part

Comment (streeve) [reply models/MFCStack.py]: Fixed

File: models/GATStack.py (hunk 1 above)
Comment (streeve) [reply models/GATStack.py:55]: Fixed

**Review by allaffa — APPROVED** [review]

**Pull request merged**